include/geode/mesh/core/detail/facet_storage.h

```diff
@@ -114,6 +114,10 @@ namespace geode
 
             void remove_facet( const VertexCycle& vertices )
             {
+                if( facet_indices_.find( vertices ) == facet_indices_.end() )
```

Comment:
1) const auto it = facet_indices_.find( vertices )
2) if
3) const auto id = *it;